### PR TITLE
GUI-2197 Alarm Detail Page

### PR DESCRIPTION
**Pull request by coyote240**
https://eucalyptus.atlassian.net/browse/GUI-2197

### CODE REVIEW

File: eucaconsole/constants/cloudwatch.py [hunk 1]
```diff
@@ -106,6 +106,7 @@
     'NetworkOut': _(u'Network out'),
     # ELB Metrics
     'RequestCount': _(u'Sum request count'),
+    'SumRequestCount': _(u'Sum request count'),
```
Comment (kamalgill) [inline eucaconsole/constants/cloudwatch.py]: @coyote240 SumRequestCount isn't an ELB metric... see http://docs.aws.amazon.com/AmazonCloudWatch/latest/DeveloperGuide/elb-metricscollected.html

**Comment by aliceh** [comment]
Retest this please

File: eucaconsole/static/css/font-awesome/font-awesome.css [hunk 2]
```diff
@@ -0,0 +1,1105 @@
+@charset "UTF-8";
```
Comment (dkavanagh) [inline eucaconsole/static/css/font-awesome/font-awesome.css]: not meant to be checked in. sorry. there's a ticket to fix this for the future.

**Comment by dkavanagh** [comment]
Other than the extra fontawesome.css, good to merge!

**Pull request merged**